var/spack/repos/builtin/packages/py-enum34/package.py

```diff
@@ -29,15 +29,13 @@ class PyEnum34(PythonPackage):
     """Python 3.4 Enum backported to 3.3, 3.2, 3.1, 2.7, 2.6, 2.5, and 2.4."""
 
     homepage = "https://pypi.python.org/pypi/enum34"
-    url      = "https://pypi.python.org/packages/source/e/enum34/enum34-1.1.6.tar.gz"
 
     version('1.1.6', '5f13a0841a61f7fc295c514490d120d0',
             url="https://pypi.python.org/packages/bf/3e/31d502c25302814a7c2f1d3959d2a3b3f78e509002ba91aea64993936876/enum34-1.1.6.tar.gz")
 
-    depends_on('python@2.4:2.8,3.3:')
+    depends_on('python@2.4:3.3.99')
```

Comment:
> Does `:3.3` include `3.3.1` or does it not?

Did some quick tests to confirm this. Version ranges are _inclusive_:
```
$ spack python
Spack version 0.10.0
Python 2.7.12, Darwin x86_64
>>> from spack.version import *
>>> Version('2.3') in VersionRange('2.4', '3.3')
False
>>> Version('2.4') in VersionRange('2.4', '3.3')
True
>>> Version('3.3') in VersionRange('2.4', '3.3')
True
>>> Version('3.4') in VersionRange('2.4', '3.3')
False
```
and _include minor versions_ as well:
```
>>> Version('2.3.9') in VersionRange('2.4', '3.3')
False
>>> Version('2.4.1') in VersionRange('2.4', '3.3')
True
>>> Version('3.3.1') in VersionRange('2.4', '3.3')
True
>>> Version('3.4.1') in VersionRange('2.4', '3.3')
False
```